src/components/user/EditProfileForm.tsx

```diff
@@ -0,0 +1,105 @@
+"use client";
```

Comment:
Moved `EditProfileForm` to components folder as it should be used across all user view.